scripts/launch-prover-service

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
I think this script is a bit risky because it's easy to trigger a deployment (for example by forgetting or misspelling the env var). This could potentially cost a lot of money. I think it would be preferable if there was an explicit argument to trigger the deployment or maybe better having a separate docker image with foundry and deployment (with for example a `-dev` suffix in the name).

Reply:
Not saying we need to fix it immediately but ideally before we deploy anything to mainnet.

Reply:
good point, let me add a `--deploy` argument just like commitment-task

Reply:
or have a env variable

Reply:
env `ESPRESSO_SEQUENCER_DEPLOY_LIGHTCLIENT_CONTRACT` added